code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1298,3 +1298,97 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	M.cultslurring = min(M.cultslurring + 3, 3)
 	M.stuttering = min(M.stuttering + 3, 3)
 	..()
+
+/datum/reagent/consumable/ethanol/triple_sec
+	name = "Triple Sec"
+	id = "triple_sec"
+	description = "A sweet and vibrant orange liqueur."
+	color = "#ffcc66"
+	boozepwr = 30
+	taste_description = "a warm flowery orange taste which recalls the ocean air and summer wind of the caribbean"
+	glass_icon_state = "glass_orange"
+	glass_name = "Triple Sec"
+	glass_desc = "A glass of straight Triple Sec."
+
+/datum/reagent/consumable/ethanol/creme_de_menthe
+	name = "Creme de Menthe"
+	id = "creme_de_menthe"
+	description = "A minty liqueur excellent for refreshing, cool drinks."
+	color = "#00cc00"
+	boozepwr = 20
+	taste_description = "a minty, cool, and invigorating splash of cold streamwater"
+	glass_icon_state = "glass_green"
+	glass_name = "Creme de Menthe"
+	glass_desc = "You can almost feel the first breath of spring just looking at it."
+
+/datum/reagent/consumable/ethanol/creme_de_cacao
+	name = "Creme de Cacao"
+	id = "creme_de_cacao"
+	description = "A chocolatey liqueur excellent for adding dessert notes to beverages and bribing sororities."
+	color = "#996633"
+	boozepwr = 20
+	taste_description = "a slick and aromatic hint of chocolates swirling in a bite of alcohol"
+	glass_icon_state = "glass_brown"
+	glass_name = "Creme de Cacao"
+	glass_desc = "A million hazing lawsuits and alcohol poisonings have started with this humble ingredient."
+
+/datum/reagent/consumable/ethanol/quadruple_sec
+	name = "Quadruple Sec"
+	id = "quadruple_sec"
+	description = "Kicks just as hard as licking the powercell on a baton, but tastier."
+	color = "#cc0000"
+	boozepwr = 30
+	taste_description = "an invigorating bitter freshness which suffuses your being; no enemy of the station will go unrobusted this day"
+	glass_icon_state = "glass_red"
+	glass_name = "Quadruple Sec"
+	glass_desc = "An intimidating and lawful beverage dares you to violate the law and make its day. Still can't drink it on duty, though."
+
+/datum/reagent/consumable/ethanol/quadruple_sec/on_mob_life(mob/living/M)
+	if(M.mind && M.mind.assigned_role in list("Security Officer", "Detective", "Head of Security", "Warden", "Lawyer")) //Securidrink in line with the screwderiver for engineers or nothing for mimes.
+		M.heal_bodypart_damage (1,1,1)
+		M.adjustBruteLoss(-2,0)
+		. = 1
+	return ..()
+
+/datum/reagent/consumable/ethanol/quintuple_sec
+	name = "Quintuple Sec"
+	id = "quintuple_sec"
+	description = "Law, Order, and Police Brutality distilled into one single elixir of JUSTICE."
+	color = "#ff3300"
+	boozepwr = 40
```

Comment:
Increase the boozepwr on this to maybe 80 or more. I like niche drinks but given the strength of this drink it needs some serious side-effects (plus you've got a seriously concentrated drink at this point). 

Plus there's something hilarious about a nigh-unkillable officer just stumbling around puking their guts out.

Reply:
I think that's an excellent balance direction. Quadruple sec is fairly mild, and is only two ~40abv drinks combined with a non-alcoholic cocktail, so I think keeping it at around 35ish boozepwr is fine (also it isn't going to make you Judge Dredd himself), but you're correct that Quintuple should make you Judge Dredd... at a price. Old Man Henderson Cop balance update incoming.

Reply:
Nitpicking / applying real world logic to this, but this drink's ABV would be roughly around 31.75%, which is around the ballpark of an old-fashioned (which surprisingly we don't have.) 

The triple-sec would be the highest ABV ingredient at around 40% ABV, so this wouldn't make sense at even this strength (since you wouldn't be increase the amount of alcohol in something by combining everything, you'd be diluting it.)

Reply:
Honestly, that's one of the reasons I had it being lower to begin with; I'm not sure if boozepwr is supposed to directly correlate with abv values, but I was roughly going off of that originally. The current update is solely a balance thing; going through and fixing all of the different drinks to have boozepwr correlating to their real world abv would be something I'd be willing to do, but I suspect that boozepwr is largely a balance factor in most drinks.